src/chapter1/isbn.hs

```diff
@@ -0,0 +1,35 @@
+module Chapter1.Isbn where
+import Data.List
+import Data.Char
+rmdash :: String -> String
```

Comment:
you are assuming that the string has at most 3 hyphens. However, the problem statement clearly states the following:

> The second part of this exercise is to modify your solution to the first part, such that it is able to “skip” hyphens anywhere in the ISBN, i.e. you can’t assume that the hyphens are in fixed/known places. The function signatures need not change.